__tests__/intergration.test.js

```diff
@@ -78,4 +78,46 @@ describe("GET /api", () => {
             }
         })
     })
-});
\ No newline at end of file
+});
+describe.only("GET /api/articles/:id/comments", () => {
+    test("Should return 200 status when passed an article ID with comments", () => {
+        return request(app).get("/api/articles/1/comments").expect(200);
+    })
+    test("Should return an object with comments property containing an array of results", () => {
+        return request(app).get("/api/articles/1/comments").expect(200).then(result => {
+            const returnObj = result.body;
+            expect(returnObj).hasOwnProperty("comments");
+            expect(Array.isArray(returnObj.comments)).toBe(true);
+        })
+    })
+    test("Each comment object in the return array should have correct properties", () => {
+        return request(app).get("/api/articles/1/comments").expect(200).then(result => {
+            const returnArray = result.body.comments;
+            for (const index in returnArray) {
+                expect(typeof returnArray[index].comment_id).toBe("number");
+                expect(typeof returnArray[index].votes).toBe("number");
+                expect(typeof returnArray[index].created_at).toBe("string");
+                expect(typeof returnArray[index].author).toBe("string");
+                expect(typeof returnArray[index].body).toBe("string");
+                expect(typeof returnArray[index].article_id).toBe("number");
+                expect(returnArray[index].article_id).toBe(1);
```

Comment:
Seeing as you're doing 
`expect(returnArray[index].article_id).toBe(1);`
then you don't really need the 
`expect(typeof returnArray[index].article_id).toBe("number");`
check. If it satisfies the toBe it must have the correct type so the type check is redundant